Match the condition in lrzip_private.h; the question is if this is really
necessary on Cygwin.

--- origsrc/lrzip-0.621/rzip.c	2015-03-10 05:13:22.000000000 -0500
+++ src/lrzip-0.621/rzip.c	2015-03-25 10:46:59.271440100 -0500
@@ -772,7 +772,7 @@ static inline void init_hash_indexes(str
 		st->hash_index[i] = ((random() << 16) ^ random());
 }
 
-#if defined(__APPLE__) || defined(__FreeBSD__)
+#if !defined(__linux)
 # define mremap fake_mremap
 
 static inline void *fake_mremap(void *old_address, size_t old_size, size_t new_size, int flags __UNUSED__)
